data/org.freedesktop.portal.Clipboard.xml

```diff
@@ -0,0 +1,150 @@
+<?xml version="1.0"?>
+<!--
+ Copyright 2022 Google LLC
+
+ This library is free software; you can redistribute it and/or
+ modify it under the terms of the GNU Lesser General Public
+ License as published by the Free Software Foundation; either
+ version 2 of the License, or (at your option) any later version.
+
+ This library is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ Lesser General Public License for more details.
+
+ You should have received a copy of the GNU Lesser General Public
+ License along with this library. If not, see <http://www.gnu.org/licenses/>.
+-->
+
+<node name="/"
+  xmlns:doc="http://www.freedesktop.org/dbus/1.0/doc.dtd">
+  <!--
+   org.freedesktop.portal.Clipboard
+   @short_description: Clipboard portal
+
+   The Clipboard portal allows clipboard control over remote desktop sessions.
+
+   This documentation describes version 1 of this interface.
+  -->
+  <interface name="org.freedesktop.portal.Clipboard">
+    <!-- 
+    EnableClipboard:
+    @session_handle: Object path for the #org.freedesktop.portal.Session object
+    @options: Options for the clipboard
+    @handle: Object path for the #org.freedesktop.portal.Request object representing this call
+
+    Enables clipboard control for the given remote desktop session.
+
+    Supported keys in the @options vardict include:
+    <variablelist>
+      <varlistentry>
+        <term>handle_token s</term>
```

Comment:
I think we can skip using `o.f.p.Request` and friends; it was a mistake it was used for `SelectDevices` and `SelectSources` in the other interfaces - the Request object / response signal is primarily used for requests that can take an indefinite amount of time due to a user query, but that is not the case for this method, which is just for setting up a request that at a later point in time will result in a user being potentially queried.